packages/network/connman/package.mk

```diff
@@ -92,7 +92,8 @@ post_makeinstall_target() {
         -e "s|^# PreferredTechnologies.*|PreferredTechnologies = ethernet,wifi,cellular|g" \
         -e "s|^# TetheringTechnologies.*|TetheringTechnologies = wifi|g" \
         -e "s|^# AllowHostnameUpdates.*|AllowHostnameUpdates = false|g" \
-        -e "s|^# PersistentTetheringMode.*|PersistentTetheringMode = true|g"
+        -e "s|^# PersistentTetheringMode.*|PersistentTetheringMode = true|g" \
+        -e "s|^# NetworkInterfaceBlacklist = vmnet,vboxnet,virbr,ifb|NetworkInterfaceBlacklist = vmnet,vboxnet,virbr,ifb,docker,veth|g"
```

Comment:
this is wrong. should be

```
s|^# NetworkInterfaceBlacklist.*|NetworkInterfaceBlacklist = vmnet,vboxnet,virbr,ifb,docker,veth|g
```

like everything else above. because if one is stupid enough to bump connman some day, and default blacklist gets changed in shipped conf. shit happens.